pkg/tests/ossm/deploy_on_infra_test.go

```diff
@@ -48,16 +49,14 @@ func TestDeployOnInfraNodes(t *testing.T) {
 		}
 
 		t.Cleanup(func() {
+			csvVersion := operator.GetCsvName(t, env.GetOperatorNamespace(), "servicemeshoperator")
```

Comment:
Maybe it's a little confusing to call the variable csvVersion but the function is getting the csvName. So, I think you can unify that to use the same, with the that who reads the test will know what happens

Reply:
yes, it is a little bit confusing ( also when the variable is added into a function that wants csvVersion :D since the csv name has format `servicemeshoperator.v2.5.0` so it is name and version at once :D ) I will change it here for better readability.